### PR TITLE
release notes for nsc

### PR DESCRIPTION
**Pull request by aricart**
This is a pull request to allow everyone to comment before we release.
The strategy is to bump the version of nsc to match the current server to facilitate the understanding that this release of nsc is only compatible with 2.2 nats-server.

The upgrade process for nsc/and environment is explained. When upgrading nsc, these release notes will be visible (these will be the release notes)

### CODE REVIEW

**Review by matthiashanel — APPROVED** [review]
LGTM

File: nsc2_releasenotes.md [hunk 1]
```diff
@@ -12,28 +12,28 @@ You can easily move between releases by doing `nsc update --version XXX`, where
...
 
-- Upgrade your nsc configurations. Use the `nsc upgrade-jwt` command. You will need the Operator's main identity key to perform the upgrade.
+- Upgrade your NSC configurations. Use the `nsc upgrade-jwt` command. You will need the Operator's main identity key to perform the upgrade.
 
-- Redistribute the updated Operator JWT, to all your servers. You can easily export the operator JWT by issuing the command. If your server is configured to a JWT operator file, make sure the new file is referenced. If the JWT is embedded into the server configuration, make sure to copy the contents of an exported operator JWT. To example the operator to a file, you can execute the command: `nsc describe operator --raw > /tmp/operator.jwt`.
+- Redistribute the updated Operator JWT, to all of your servers. You can easily export the operator JWT by issuing the command. If your server is configured to a JWT operator file, make sure the new file is referenced. If the JWT is embedded into the server configuration, make sure to copy the contents of an exported operator JWT. To example the operator to a file, you can execute the command: `nsc describe operator --raw > /tmp/operator.jwt`.
```
Comment (gcolliso) [inline nsc2_releasenotes.md:21]: @aricart The sentence 'To example the operator to a file...'  Is 'example' the correct word here?

**Review by wallyqs — APPROVED** [review]
LGTM, just same comment on releasing nsc v1.0.0 (from same tag as v0.5.0) first then branch of that if needed for fixes for setups that not yet upgraded to nsc v2.2.

File: nsc2_releasenotes.md (hunk 1 above)
Comment (aricart) [reply nsc2_releasenotes.md:21]: sigh - no - "For example, to save your operator to a file..."

**Comment by aricart** [comment]
> LGTM, just same comment on releasing nsc v1.0.0 (from same tag as v0.5.0) first then branch of that if needed for fixes for setups that not yet upgraded to nsc v2.2.

Won't do it - the folks at goreleaser changed the configuration and invalidated one of the config values, which fails the release. I will keep the tag for giggles, but we are going to stand by the 0.50.0

**Comment by gcolliso** [comment]
lgtm

**Review by ColinSullivan1 — APPROVED** [review]
LGTM.  Thanks @aricart !

**Comment by aricart** [comment]
closing as these notes are part of the release

**Pull request closed without merging**